src/setup_payload/tests/TestAdditionalDataPayload.cpp

```diff
@@ -221,6 +223,16 @@ void TestParsingAdditionalDataPayloadWithLongRotatingDeviceId(nlTestSuite * inSu
                                               resultPayload) == CHIP_ERROR_INVALID_STRING_LENGTH);
 }
 
+void TestParsingAdditionalDataPayloadWithShortRotatingDeviceId(nlTestSuite * inSuite, void * inContext)
+{
+    chip::SetupPayloadData::AdditionalDataPayload resultPayload;
+    NL_TEST_ASSERT(inSuite,
+                   ParseAdditionalDataPayload(kAdditionalDataPayloadWithShortRotatingDeviceId,
+                                              strlen(kAdditionalDataPayloadWithShortRotatingDeviceId),
+                                              resultPayload) == CHIP_NO_ERROR);
+    NL_TEST_ASSERT(inSuite, strcmp(resultPayload.rotatingDeviceId.c_str(), kShortRotatingDeviceId) == 0);
```

Comment:
my bad, i thought you mean short rotating id in the additionalDataPayload itself not separately, Thanks for adding this test.